code/game/machinery/autolathe.dm

```diff
@@ -268,8 +261,28 @@
 		busy = FALSE
 		return TRUE
 
+	if(panel_open)
+		balloon_alert(user, "close the panel first!")
+		return FALSE
+
 	return ..()
 
+/obj/machinery/autolathe/attackby_secondary(obj/item/weapon, mob/living/user, params)
+	if (busy)
+		to_chat(user, "<span class=\"alert\">The autolathe is busy. Please wait for completion of previous operation.</span>")
```

Comment:
change this to span defines.

Reply:
its either that or balloon alerts, up for both

Reply:
balloons are better

Reply:
```suggestion
		balloon_alert(user, "it's busy!")
```

Reply:
will wrap it up tomorrow